libs/composer/vendor/composer/autoload_static.php

```diff
@@ -4554,6 +4554,7 @@ class ComposerStaticInit2fffdf922cf8fdbf1f62eec345993c83
         'ilExerciseManagementGUI' => __DIR__ . '/../..' . '/../../Modules/Exercise/classes/class.ilExerciseManagementGUI.php',
         'ilExerciseMemberTableGUI' => __DIR__ . '/../..' . '/../../Modules/Exercise/classes/class.ilExerciseMemberTableGUI.php',
         'ilExerciseMembers' => __DIR__ . '/../..' . '/../../Modules/Exercise/classes/class.ilExerciseMembers.php',
+        'ilExerciseMembersFilter' => __DIR__ . '/../..' . '/../../Modules/Exercise/classes/class.ilExerciseMembersFilter.php',
```

Comment:
This doesn't belong here, right?

Reply:
Seems to be forgotten in a prior commit by another dev. I always rebuild the autoload classmap when I introduce new files, which I have done here. But of course my new files are completely irrelevant for the classmap because they are located under `./src`.